app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/ezyoutube.html.twig

```diff
@@ -0,0 +1,9 @@
+<div{% if params.align is defined %} style="text-align: {{ params.align }};"{% endif %}>
+    <iframe
+        type="text/html"
+        width="{{ params.width }}"
+        height="{{ params.height }}"
+        src="{{ params.video_url|replace({'https://youtu.be/' : 'https://www.youtube.com/embed/'}) }}?autoplay={{ params.autoplay == 'true' ? 1 : 0 }}"
```

Comment:
I just tried the PR. `URL` is kind of confusing, because I copied / pasted the URL of the browser, not the one with the `SHARE` button.
We can maybe change `Video URL` by `Video ID`. What do you think?

Reply:
It just requires some doc. We've discussed Video ID vs some URL and came to conclusion that an average user won't understand what is video ID and also won't be able to select properly ID portion to be copied. Unfortunately it's not possible to embed video copying direct URL.

You have "Share" button below video which contains valid URL:
![image](https://user-images.githubusercontent.com/7099219/37620451-8e497e96-2bbc-11e8-9988-7d6b07159395.png)

But yes, if you believe Video ID is better - we can change it, after all Custom Tags are... custom :)

Reply:
I totally understand the approach. Let's keep it like that. :)
Thanks @alongosz

Reply:
By the way, at some point, I remember thinking of an extra label for attributes, that would be filled using translations, like `custom_tags.foo.attributes.bar.helptext`. For such cases, it would allow us to document this item a little bit in the dialog.